apps/labs/posts/python-packaging-where-to-next.md

```diff
@@ -0,0 +1,352 @@
+---
+title: 'Python packaging & workflows - where to next?'
+published: January 10, 2023
+author: ralf-gommers
+description: 'Potential solutions for pain points when dealing with native code; what needs unifying in the Python packaging space, and how should that be approached? '
+category: [Python packaging]
+featuredImage:
+  src: /posts/python-packaging-where-to-next/a-custom-image-I-think.png
+  alt: 'todo.'
+hero:
+  imageSrc: /posts/python-packaging-where-to-next/hero-image.png
+  imageAlt: 'todo.'
+---
+
+A lot is going on in Python packaging land. Two weeks ago I released
+[the pypackaging-native website](https://pypackaging-native.github.io/), which
+documents extensively what the most important issues are with packaging Python
+projects that contain code that needs to be compiled. It's something I had wanted to have for a long time, and I sincerely hope thatit'll help improve the quality of Python packaging design discussion.
+Right before that release, there was a large discussion on Discourse -
+[Wanting a singular packaging tool/vision ](https://discuss.python.org/t/wanting-a-singular-packaging-tool-vision/21141)
+- about future wishes and big picture packaging topics/changes. A second big
+picture thread,
+[Python Packaging Strategy Disccusion - Part 1](https://discuss.python.org/t/python-packaging-strategy-discussion-part-1/22420)
+is active right now (and the title implies there'll at least be a "strategy -
+part 2" one).
+
+`pypackaging-native` content on purpose does not go into detail on solution
+directions. Real-world problems are, well, real - they're hard to argue with.
```

Comment:
By "they're hard to argue with", are you meaning that you chose that scope for the website so as to try to avoid having the site be a locus of big discussions? If so, that's not especially clear.

Reply:
Exactly. Let me try to rephrase.